gwpy/signal/qtransform.py

```diff
@@ -532,6 +533,8 @@ def interpolate(self, tres=.001, fres="<default>",
         dtype = self.energies[0].dtype
         # build regular Spectrogram from peak-Q data by interpolating each
         # (Q, frequency) `TimeSeries` to have the same time resolution
+        if tres == "<default>":
+            tres = abs(Segment(outseg)) / 1000
```

Comment:
```suggestion
            tres = abs(Segment(outseg)) / 1000.
```

Reply:
@duncanmmacleod, this is done in the latest commit.